--- /usr/share/rhn/up2date_client/debUtils.py	2015-08-20 10:40:51.034509235 +0200
+++ /root/debUtils.py	2015-08-20 10:45:19.650509235 +0200
@@ -25,13 +25,13 @@

 def parseVRE(version):
     epoch = ''
-    release = '0'
-    if version.find(':') != -1:
-        epoch, version = version.split(':')
-    if version.find('-') != -1:
-        tmp = version.split('-')
-        version = '-'.join(tmp[:-1])
-        release = tmp[-1]
+    version_tmpArr = version.split('-')
+        if len(version_tmpArr) == 1:
+            version = version
+            release = "X"
+        else:
+            version = version_tmpArr[0]
+            release = version_tmpArr[1]
     return version, release, epoch

 def installTime(pkg_name):