--- xmlrpclib.py.org	2015-08-20 14:37:38.202407951 +0200
+++ xmlrpclib.py.mod	2015-08-20 14:41:03.982407951 +0200
@@ -655,8 +655,8 @@
         f(self, value, write)
 
     def dump_nil (self, value, write):
-        if not self.allow_none:
-            raise TypeError, "cannot marshal None unless allow_none is enabled"
+        #if not self.allow_none:
+        #    raise TypeError, "cannot marshal None unless allow_none is enabled"
         write("<value><nil/></value>")
     dispatch[NoneType] = dump_nil
 
